packages/react-components/react-nav-preview/library/src/components/AppItemStatic/AppItemStatic.types.ts

```diff
@@ -12,6 +20,11 @@ export type AppItemStaticProps = ComponentProps<AppItemStaticSlots> & {};
 /**
  * State used in rendering AppItemStatic
  */
-export type AppItemStaticState = ComponentState<AppItemStaticSlots>;
-// TODO: Remove semicolon from previous line, uncomment next line, and provide union of props to pick from AppItemStaticProps.
-// & Required<Pick<AppItemStaticProps, 'propName'>>
+export type AppItemStaticState = ComponentState<AppItemStaticSlots> & {
+  /**
+   * The size of the Nav
+   *
+   * @default 'medium'
+   */
+  size: NavSize;
```

Comment:
nit: Could pick this directly from context, i.e.
ComponentState<AppItemStaticSlots> & Pick<NavContextType, 'size'>;